### ISSUE TITLE
Some issues

### ISSUE THREAD

**Issue by Misio12320** [Pokechu22/WorldDownloader#92]
When will the chests are fixed? How to make it to download the world with more than 10 mobs, villagers, sheeps etc. The world downloaded in 3 MB congrats . It looks that https://gyazo.com/6a4624899600daeb965f7b9106ceb2eb why the built isnt all... What I need to do to get the full download world which is larger...

**Comment by Pokechu22** [comment]
You need to go around the whole world to save it all. Only the areas you have gone to/seen are saved (just as you need to containers to save them). When a chunk is unloaded then it will be saved.

I don't have an ETA for when chests will be fixed completely.  Note that they should _mostly_ save fine in simple designs; it is only when there are multiple regular chests next to each other that problems happen.

**Comment by Misio12320** [comment]
So I need destroy the chests and put them outta my house? Well. Or...

**Comment by Pokechu22** [comment]
No, you do not need to break them. The only problem is when you have two large chests that touch each other. And even then, it just means that the items in those chests will not be saved/will be in the wrong position, but the rest of the saved world will still work fine. (Consider saving a screenshot of each chest if you would like to have a record of it)

**Comment by Misio12320** [comment]
So I can normal have 2 large chests but 2 blocks on right left or middle next chests? What about up chest?

**Comment by Misio12320** [comment]
Please make a video tutorial or full info what every option do.... So I start the server I fly everywhere and the chunks which i fly it Will be downloaded? What about Render distsnce? I go on 1 chunk 16x16 or and I idk

**Comment by V-DeVito** [comment]
Wszystko, co jest widoczne dla odtwarzacza, zostanie pobrane. Przekręć odległość renderowania aż do maksymalnego zasięgu. Jeśli chcesz pobrać duże kawałki ziemi, idź, dopóki nie pokryjesz całego obszaru. Jeśli chodzi o skrzynie, dwie duże skrzynie obok siebie nie będą działać. Wszystko inne będzie dobrze.

**Comment by Misio12320** [comment]
Ouh. How can I fill this with the own seed? https://gyazo.com/9ed54efebc871f9b7a7add5a90d71df3 also fill outsite the blocks I think....
Well I understand this language how did you known if Im Polish? Nick you saw my

**Comment by Pokechu22** [comment]
The different options are documented on [the wiki](https://github.com/Pokechu22/WorldDownloader/wiki/Player-overrides-GUI), though that information is a bit old.  Documentation also appears in-game when you hover over the button.

Chests above each other are fine.  The easiest way to explain it is that WDL behaves as if chests were in 1.12, which has different rules for where chests can be placed.  Play with chests in creative in 1.12 and it should be a lot more obvious.

**Comment by Misio12320** [comment]
I put this seed into the download and i pressed done and starter download... Hmm

**Comment by Pokechu22** [comment]
The world generator option is actually currently broken; it'll be fixed in the next update to the mod.  (That's also why it's generating superflat plains instead of void)

**Comment by Pokechu22** [comment]
I've released [version 4.0.3.1](https://github.com/Pokechu22/WorldDownloader/releases/tag/v4.0.3.1), which fixes the large chest issue and the generator issue.

**Comment by Misio12320** [comment]
Thanks for the info... Please answer on my ''suggestion'' I mean download world and the chunks outside border doesnt download or select xz1 xz2 and yeah

**Comment by Misio12320** [comment]
i cant join 1.13 to 1.13.1 on 4.0.3.1...

**Comment by Misio12320** [comment]
Again chests are  empty half and full without together and together... Great  job seeds are work :D thats progress
